swim/ping_request_sender.go

```diff
@@ -39,17 +39,6 @@ type pingRequest struct {
 	Changes           []Change `json:"changes"`
 }
 
-// newPingRequest creates a new ping request to the specified target.
```

Comment:
This patch is literally a reversal of a patch I made which introduced a constructor for `pingRequest`.

Personally, I find the lack of constructor here more difficult to understand, as without it I have to concern myself with every field of the `pingRequest`, even though only one or two fields should be set. `Source`, `SourceIncarnation`, `Checksum` should never be set manually, right? They are always going to refer to the node's address, incarnation number and checksum respectively.